### ISSUE TITLE
Fallback on using CURL or wget if perl LWP::UserAgent is not available

### ISSUE THREAD

**Issue by gwarf** [CESNET/pakiti-client#13]
Fallback on using CURL or wget if perl LWP::UserAgent is not available, as apparently some sites are not having it by default, 
preventing to run pakiti correctly from CVMFS.

It may be done based on https://github.com/CESNET/pakiti-client/commit/f5e2dea7b0325293261f150552bd9660755634f6

Another solution may be to include the required dependencies in CVMFS, but it may cause issues depending on the OS running the client.

**Comment by beckerr-rzht** [comment]
One could turn the absence of LWP::Agent into a soft error using this:
```diff
--- -   2022-11-30 08:28:08.448968487 +0100
+++ /usr/local/bin/pakiti-client        2022-11-30 08:25:48.163287530 +0100
@@ -17,7 +17,7 @@
 use FindBin qw($Bin $Script);
 use Getopt::Long qw(GetOptions);
 use Pod::Usage qw(pod2usage);
-use LWP::UserAgent ();
+eval { require LWP::UserAgent; LWP::UserAgent->import; };
 
 #
 # constants
```
Then the error would only be triggered when `--url` is actually used:
```bash
$ pakiti-client >/dev/null                       
report successfully printed

$ pakiti-client --url https://foobar/pakiti/feed/
Can't locate object method "new" via package "LWP::UserAgent" (perhaps you forgot to load "LWP::UserAgent"?) at /usr/local/bin/pakiti-client line 389.
```

The feeder command without `--url` could look like this:
```
$ pakiti-client | curl -sS --data-binary @- --url https://foobar/pakiti/feed/
report successfully printed
OK
```

**Comment by gwarf** [comment]
Indeed there are workarounds, but for our usage we need to have it working in an as straightforward way as possible, without having to rely on users managing the workaround themselves.
So if pakiti client itself could catch the missing package and then fallback on curl/wget (as it was doing before) it would cover most of the cases, in some kind of graceful degradation.

**Comment by gwarf** [comment]
Fixed by #15
